src/index.tsx

```diff
@@ -73,11 +72,29 @@ var createWebmarkFolder = (): void => {
     )
 };
 
-var getCurrentUrl = (): string => {
-    return 'https://current.url.com';
+// type checker for proper string type url
+let typeCheckString = (text: string | undefined): string => {
+    if (text != undefined) {
```

Comment:
This is a personal preference, but I avoid `if(negative)` because that makes the `else` a double negative and therefore confusing.

Reply:
Good point. Submit a change request and I will fix the syntax.